addons/sale/static/src/scss/product_configurator.scss

```diff
@@ -122,6 +122,11 @@ label.css_attribute_color.css_not_available {
     .o_total_row {
         font-size: 1.2rem;
     }
+
+    img.variant_image {
+        height: 128px;
+        width: 128px;
```

Comment:
We can't hardcode both width and height on the image, because when using the image field (= big size) it might not be a square, so it will distort the image.

What should be done ideally is the parent to be the square, and the image to adapt into it by keeping its ratio (you can check what I did in master with the carousel preview on website_sale).
The problem is that there is no suitable parent for that here.

So either we can find other CSS rules, for example by not forcing a square.
Or we need to change the JS to use the image_medium when showing the image inside the modal.

Finding a way to use image_medium would be the best way, because that's the minimal download size for the user.